src/frontend/magnify/src/services/array/array.helper.ts

```diff
@@ -0,0 +1,18 @@
+export class ArrayHelper {
+  public static findElementIndex<T>(array: T[], element: T): number | null {
+    const index = array.findIndex((arrayElement: T) => {
+      return arrayElement == element;
```

Comment:
Unfortunately this is not working with objects.
If this is not a use case here, I'll just update the function signature to forbid to type T as an object.
```ts
findElementIndex<T extend number, string, boolean ...>( ... ) { ... }
```

Reply:
agreed